contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -362,6 +362,27 @@ def val_for_column(column, clip):
             return otio.opentime.to_timecode(
                 clip.source_range.end_time_exclusive(), fps
             )
+        elif column == "ASC_SOP":
+            asc_sop = clip.metadata.get("cdl", {}).get("asc_sop", {})
+            offset = asc_sop.get("offset", None)
+            power = asc_sop.get("power", None)
+            slope = asc_sop.get("slope", None)
+            asc_sop_arr = [offset, power, slope]
+            if None in asc_sop_arr:
+                return clip.metadata.get("ALE", {}).get(column)
+            asc_sop = ""
+            for i in range(3):
+                asc_sop += "("
+                asc_sop += str(asc_sop_arr[i][0]) + " "
+                asc_sop += str(asc_sop_arr[i][1]) + " "
+                asc_sop += str(asc_sop_arr[i][2]) + ")"
+            return asc_sop
+        elif column == "ASC_SAT":
+            asc_sat = clip.metadata.get("cdl", {}).get("asc_sat", None)
+            if clip.metadata.get is not None:
```

Comment:
This is checking that the `get` method `is not None` on the clip's `metadata` dictionary - that will always be `True` since methods are not `None`.
Did you mean to check `if asc_sat is not None`?